.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,7 +17,7 @@ The following tradeoffs were made:
 The following alternatives were considered:
 
 Links to places where the discussion took place: <!-- optional: slack, other GH issue, mailinglist, ... -->
--
+- 
```

Comment:
I'm going to presume that you want to remove the underlined heading for the 'Links to places ...' line and keep it as normal text?

I think you'll want to add a new line between the text and the hyphen, otherwise I think it is being treated as heading markup. btw is the hyphen for a bullet point?

Reply:
The problem was that "Links to …" was recognized as a title.

Adding the space after the dash makes it a list.

I'll add the newline nevertheless

Reply:
I've actually removed it.